services/gcp/container/serviceAccounts.yml

```diff
@@ -20,7 +20,7 @@ privileges:
       Creating a service account by itself does not represent a security risk.
       Service accounts need to be granted permissions via Roles.
   createToken:
-    risks: [escalation:privilege]
+    risks: [escalation:lateral, escalation:privilege]
```

Comment:
This is only lateral escalation, right? What's the privilege escalation?

Reply:
I was thinking that it depends on what permissions the service account has. It may escalate your existing privileges or allow lateral movement.
But perhaps it makes more sense to categorize as lateral only since most of the time that will be the case.